src/Angor/Avalonia/Angor.Wallet/Angor.Wallet.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFramework>net9.0</TargetFramework>
```

Comment:
We were having issues with NBitcoin package and .net 9 this might come up as a problem when you plug in the shared project

Reply:
Oh... Could you provide more details about the NBitcoin issues with .NET 9? I wasn't aware of any specific incompatibilities and it would help me better understand what we might need to address. We can switch to .NET 8 if needed, but I would like to keep current due to the performance improvements that were introduced lately :)

Reply:
I think the issue was with the latest version of NBitccoin not dotnet9, I recall the issue with dotnet9 was packaging the blazor application did not work well.
Here is the pr changes with NBitcoin that we made to make it work

I think it is not an issue for avalonia packaging and it will be backwards compatible so it is ok